dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -145,7 +145,7 @@
     @OrderBy("id")
     private SortedSet<FileFormat> outputFileFormats = new TreeSet<>();
 
-    @OneToMany(fetch = FetchType.LAZY, orphanRemoval = true, cascade = CascadeType.ALL)
```

Comment:
Can lead to a performance hit by getting validation info whether we need it or not. 
But this does narrow down the possible causes. 
You probably want to find out which endpoint needs validation and fetch the validations with something like `Hibernate.initialize()` or just iterate through then (i.e. version.validations.iterator() and do a no-op)

Alternatively, some endpoints already have the option to grab validations. 
The UI could use one of these instead or use the option https://github.com/dockstore/dockstore/blob/1.7.0-rc.1/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java#L436